### ISSUE TITLE
[Mod Compatibility]: Abyssalcraft

### ISSUE THREAD

**Issue by TeetoKaziim** [Shovinus/ChopDownUpdated#33]
https://gist.github.com/TeetoKaziim/1360fabb5772e11c503dde94a73f6c4c

Have I taken a moment to say just how much I appreciate this mod? It's like, such a perfect balance point between like, making every axe a lumber axe, and the dumb problem where an oak tree grows large in the early game and you just GROAN because you're gonna have to erect an entire scaffolding structure just to chop it down. Or like, stair-way-ing up a jungle or spruce 2x2 tree? Ugh.

This is in the sweet spot, in terms of play balance: Your high-tier lumber-axe/terra-truncator tools are still super valuable and relevant, but early game logging is just SO much less annoying. And, it's VERY like, emotionally satisfying, seeing a big ol tree come tumbling down.

So like, super big thankyou.

**Comment by Shovinus** [comment]
No worries at all, that is the reason I updated it, the original idea was a simple a smart solution to an annoying issue with minecraft trees, but one that was not really implemented too well.
I just took the principle, made it more flexible, more intelligent in its tree detection and more efficient in its movement methods.
This has led to what I believe is one of the best tree detection algorithms for universal trees around.
If I don't have it in a pack I end up having to add it most of the time because it ruins the immersion when it is not on for me now. 
It is not even just because I made it, I just can't imagine playing without it any more.
